phoenix-core/src/main/java/org/apache/phoenix/mapreduce/index/IndexTool.java

```diff
@@ -510,7 +542,12 @@ private Job configureJobForServerBuildIndex() throws Exception {
 
             PhoenixConfigurationUtil.setIndexToolDataTableName(configuration, qDataTable);
             PhoenixConfigurationUtil.setIndexToolIndexTableName(configuration, qIndexTable);
-
+            if (verify) {
+                PhoenixConfigurationUtil.setVerifyIndex(configuration, true);
+            }
```

Comment:
what if both set?

Reply:
if both "verify" and "only-verify" are set then as the help for the command says the only-verify option will be ignored: "To verify every data table row has the corresponding index row with the correct content (without building the index table). If the verify option is set then the only-verify option will be ignored");